pkg/utils/k8sutil.go

```diff
@@ -53,7 +53,8 @@ import (
 
 const (
 	controllerImage = "bitnami/kubeless-controller"
-	kafkaImage      = "wurstmeister/kafka"
+	kafkaImage      = "bitnami/kafka"
```

Comment:
Yeah. @jbianquetti-nami please help to tag the images. I will add follow up commit for that.